### ISSUE TITLE
The edit window has two scroll bars

### ISSUE THREAD

**Issue by mariobehling** [fossasia/susi_skill_cms#1381]
Click edit and view the double scroll bars.

![screenshot from 2018-07-31 18-10-41](https://user-images.githubusercontent.com/1583873/43453651-2ecfc926-94ed-11e8-9fa6-cc37cae73714.png)

**Comment by anshumanv** [comment]
I'll do this.

**Comment by akshatnitd** [comment]
This has been fixed IMO by @hkedia321 . But the build for merged PRs have failed, due to which we are unable to see the changes.

**Comment by hkedia321** [comment]
Yes this has been fixed in https://github.com/fossasia/susi_skill_cms/pull/1374, but due to failed build on the master branch, the changes are not reflected. Lets wait for the PR https://github.com/fossasia/susi_skill_cms/pull/1414 to be merged and the build to be passed.

**Comment by anshumanv** [comment]
Okay yes, thanks for informing. :+1:
